centreon/packages/ui/src/Listing/index.tsx

```diff
@@ -125,6 +125,9 @@ const useStyles = makeStyles<StylesProps>()(
       width: '100%'
     },
     table: {
+      '.listingHeader > div > div:first-of-type': {
+        paddingLeft: theme.spacing(1.5)
+      },
       '.listingHeader div div': {
         backgroundColor: theme.palette.background.listingHeader,
```

Comment:
those selectors aren't very precise. If the structure change, we will create a regression. 
Is there no way to pass down a class or to use a more semantic elements ?

Reply:
the class is lisitingHeader!

Reply:
> Maybe you need to show me, but I believe the feature is not right... or there is something I'm missing. the feature is not supposed to work like that.
> 
> you have a check button that trigger a check on click. you have an adjacent button with an arrow that deploy an option list by opening the option list, you select which method the first button should trigger, and the button text changes. But in any way the click on an option should trigger a check if I'm not mistaken

this is a list , displayed by the button , i dont understand why that so confusing

Reply:
@Noha-ElAbrouki yes exactly, that's not the feature.

Reply:
Regarding the css selector, it select : 
the first element of type div, direct child of a div, direct child of an element with a class 'listingHeader'
all I'm saying is it would be more future proof to create a class for the element you are tryin to target, instead of relying on dom structure. That way, if the structure change in the future, you don't have to change your selector.
otherwise you expose yourself to bugs.

Reply:
okey i see , i sent the actions directely when the user click on the option , you r right

Reply:
> yes you are right , but I did not understand you at the beginning when you said that the feature is not implemented, to understand you must specify what does not work, and not just saying the feature does not work, to take the remark into consideration, and for others make suggestions If it's possible , not a global feedback

> Maybe you need to show me, but I believe the feature is not right... or there is something I'm missing. the feature is not supposed to work like that.
> 
> you have a check button that trigger a check on click. you have an adjacent button with an arrow that deploy an option list by opening the option list, you select which method the first button should trigger, and the button text changes. But in any way the click on an option should trigger a check if I'm not mistaken

@Noha-ElAbrouki yes sorry, I described the behavior here the best I could, but if you need help, we can pair program this together.